extensions/jsch/deployment/src/main/java/io/quarkus/jsch/deployment/JSchProcessor.java

```diff
@@ -72,4 +80,14 @@ ReflectiveClassBuildItem reflection() {
                 "com.jcraft.jsch.UserAuthPassword",
                 "com.jcraft.jsch.UserAuthPublicKey");
     }
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FeatureBuildItem.JSCH);
```

Comment:
This extension was purposely designed to not produce a FeatureBuildItem since we didn't want to appear in the initialization logs

Reply:
That's good to know, thanks @gastaldi. This was part of a test that didn't produce the expected effect, so it would probably have been removed anyway.